### ISSUE TITLE
How to show the label of the result

### ISSUE THREAD

**Issue by Zynist** [Vladkryvoruchko/PSPNet-Keras-tensorflow#78]
how can i got the result of image as text with label and percentage of each label?
